concent_api/core/message_handlers.py

```diff
@@ -444,75 +459,143 @@ def handle_send_force_subtask_results_response(request, client_message):
     return HttpResponse("", status = 202)
 
 
+def sum_payments(payments: List[Union[ForcedPaymentEvent, BatchTransferEvent]]):
```

Comment:
Same here (please check my comment below first), with `mock` backend there will be also list of dicts.

Reply:
As above I change mock backend to return null values